src/app/shared/progress/progress.component.ts

```diff
@@ -79,7 +79,7 @@ export class ProgressComponent {
 
     if (event.target === this.el.nativeElement) {
       this.onBasicMousemove(event);
-    } else if (event.target === this.el.nativeElement.querySelector('scrub-detector')) {
+    } else if (event.target === this.el.nativeElement.querySelector('.scrub-detector')) {
```

Comment:
Issue #1: Since 'scrub-detector' is a class, this call to `querySelector()` was always returning `null`, so `onFancyMousemove()` would never be called.